LibreNMS/Device/YamlDiscovery.php

```diff
@@ -245,11 +247,17 @@ public static function getValueFromData($name, $index, $discovery_data, $pre_cac
         }
 
         if (isset($discovery_data['oid']) && ! is_array($discovery_data['oid']) && isset($pre_cache[$discovery_data['oid']][$index]) && isset($pre_cache[$discovery_data['oid']][$index][$name])) {
-            return $pre_cache[$discovery_data['oid']][$index][$name];
+            $value = $pre_cache[$discovery_data['oid']][$index][$name];
+            \Log::debug("Using $value from discovery data for $name");
```

Comment:
If we are going to log all these, we could make it a little more useful by mentioning the key structure of the data selection too.